update install documentation for specific version (#567)
onflow/flow-cli: docs/install.md

To update, simply re-run the installation command above.

> Currently, there is no ability to install an earlier version of flow-cli on Homebrew.

Reviewer: ```suggestion
It is currently not possible to install earlier versions of the Flow CLI with Homebrew.
```

To update, simply re-run the installation command above.

> To install a specific version of flow-cli, the command below can be used (example below is to install flow-cli version v0.33.3):

Reviewer: ```suggestion
### Install a specific version

To install a specific version of Flow CLI, append the version tag to the command (e.g. the command below installs CLI version v0.33.3).
```
